### PR TITLE
DEVELOPER-1248 Index 'Prerequisites', 'tag', 'quickstart id' and 'commit id' when indexing quickstarts

### PR DESCRIPTION
**Pull request by Dantheman720**
Final testing

This change requires: https://github.com/awestruct/aweplug/pull/99

### CODE REVIEW

**Comment by Dantheman720** [comment]
retest this please

**Comment by Dantheman720** [comment]
This change requires: https://github.com/Dantheman720/aweplug/tree/DEVELOPER-1248

**Comment by Dantheman720** [comment]
retest this please

File: Gemfile [hunk 1]
```diff
@@ -26,8 +26,7 @@ gem 'git', '~> 1.2.5'
 gem 'oily_png', '~> 1.1.1'
 gem 'nokogiri', '~> 1.5.10'
 gem 'therubyracer', platforms: :ruby, require: linux_only('therubyracer')
-gem 'aweplug', github: 'awestruct/aweplug'
```
Comment (paulrobinson) [inline Gemfile]: Don't delete this line, just comment it out. It makes it harder to revert back to the master branch if we need to go look up the exact syntax.

Comment (Dantheman720) [inline Gemfile]: Sorry about that, I can repush with it added back in.

File: Gemfile.lock [hunk 2]
```diff
@@ -1,4 +1,24 @@
 GIT
+  remote: git://github.com/Dantheman720/aweplug.git
+  revision: b73a61660f407eb8c7e76d69c395ab06aed8d0c9
```
Comment (paulrobinson) [inline Gemfile.lock]: You are not testing with the right commit. You should have this one: https://github.com/Dantheman720/aweplug/commit/21591e79452685ee3728b2d23ec885e811e1f75f

**Pull request merged**